client/app/src/views/landing.html

```diff
@@ -1 +1,12 @@
-<h1>Hello from Landing</h1>
+
+<!-- Template for Landing -->
+
+  <div class="container" ng-controller="ScreenshotController">
```

Comment:
We're specifying the controller in the ui-router. Right now, you can reference the controller as `screenshotsCtrl`. E.g. `{{ screenshotsCtrl.data.screenshots }}`. See `app.js` for more info.